priv/repo/migrations/20160802030642_create_note.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Notox.Repo.Migrations.CreateNote do
+  use Ecto.Migration
+
+  def change do
+    create table(:notes) do
+      add :title, :string
+      add :description, :string
+
+      timestamps
```

Comment:
Yep my bad. It was having trouble migrating forward with the null option in there, but it must have been something I was doing wrong.